### DISCUSSION TITLE
调用interfaces里面的功能呢？？

### DISCUSSION THREAD

**Discussion by yuhaiwan5921** [kbengine/kbengine#1554] [Q&A] [answered]
请教大神，我要如何配置才能，调用interfaces里面的功能呢？？

**Comment by Wandar** [comment, marked as answer]
interface没啥用,功能用baseapp做

**Reply by yuhaiwan5921** [reply]
我想实现手机和第三方登录及第三方支付功能，我看示例里面的interface里面的kbemain.py里面有实现，但是我不知道怎么调用。
